projects/ngrx.io/content/guide/migration/v15.md

```diff
@@ -65,95 +68,53 @@ const mySelector = createSelector(
 (mySelector.projector as any)()
 ```
 
-The projector method has become strict for selectors with props
-
-BEFORE:
-
-You could pass any arguments to the projector method
-
-```ts
-const selector = createSelector(
-  selectString, // returning a string
-  selectNumber, // returning a number
-  (s, n, prefix: string) => {
-    return prefix + s.repeat(n);
-  }
-)
-
-// you could pass any argument
-selector.projector(1, 'a', true);
-```
-
-AFTER:
-
-```ts
-const selector = createSelector(
-  selectString, // returning a string
-  selectNumber, // returning a number
-  (s, n, prefix: string) => {
-    return prefix + s.repeat(n);
-  }
-)
+### @ngrx/effects
 
-// this throws
-selector.projector(1, 'a', true);
-// this does not throw because the arguments have the correct type
-selector.projector(1, 'a', 'prefix');
-```
+#### Removal of @Effect
 
-### @ngrx/effects
+The `@Effect` decorator is removed in favor of [`createEffect`](https://ngrx.io/api/effects/createEffect).
+This change also means that the ESLint rules @ngrx/no-effect-decorator-and-creator and @ngrx/no-effect-decorator are removed.
 
-The @Effect decorator is removed
+> A migration was added (in the v13 release) to upgrade existing codebases to the `createEffect` function.
 
 BEFORE:
 
-Defining an effect is done with @Effect
+An effect is defined with the `@Effect` decorator.
 
 ```ts
 @Effect()
 data$ = this.actions$.pipe();
 ```
 
-A migration was added in the v13 release to upgrade existing codebases to the `createEffect` function.
-
 AFTER:
 
-Defining an effect is done with createEffect
+You need to define an effect with `createEffect`.
 
 ```ts
 data$ = createEffect(() => this.actions$.pipe());
 ```
 
-The signature of `provideEffects` is changed to expect a
-spreaded array of effects.
+### @ngrx/router-store
 
-BEFORE:
+#### Title
 
-`provideEffects` expected the effects to be passed as an array.
```

Comment:
This is removed because #3638 re-allows this to keep the signatures the same for EffectsModule and provideEffects